README.md

```diff
@@ -1,3 +1,19 @@
 # alpaca-albatross-practicum-team1
 
-## Back End Repository
\ No newline at end of file
+## Back End Repository
+
+### Set up
+
+npm install
+
+### npm run dev
+
+[http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+### dependencies
+
+/node_modules
+nodemon
+mongoose
+dotenv
+express
```

Comment:
Remove this, I dont think its necessary to list the dependencies in the readme. Thos should just be explaining how to set the project up plus a description if we want